queries.md

```diff
@@ -6,27 +6,70 @@
 
  - **`query`**: {name: 'Facebook'}
  
+
  ## 2. Find all the companies which **category_code** is 'web'. Retrive only their `name` field:
 
  - **`query`**: {category_code: 'web'}
  - **`projection`**: {name: 1, _id: 0}
 
+
 ## 3. Find all the companies named "Twitter", and retrieve only their `name`, `category_code` and `founded_year` fields.
 
+- **`query`**:{name: 'Twitter'}
+- **`projection`**: {$and: [{category_code: 1}, {founded_year: 1}]}
+
+
 ## 4. Find all the companies who have `web` as their **category_code**, but limit the search to 50 companies.
 
+- **`query`**: {category_code: 1}
```

Comment:
In this case you had to do {category_code: 'web'} to find all the companies with web in the category code